package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "import-meta-resolve",
-  "version": "2.2.2",
+  "version": "3.0.0",
```

```suggestion
  "version": "2.2.2",
```

I’ll do this in a separate commit!